pages/library/_viewId.vue

```diff
@@ -30,25 +30,11 @@
         <skeleton-card v-for="n in 24" :key="n" />
       </v-col>
     </v-row>
-    <dynamic-scroller
```

Comment:
I think we should keep a virtual scroller until we have a custom replacement.

With this removed, a library of ~1200 items freezes the entire client for around 20-25 seconds on a pretty beefy config (AMD Threadripper 2950X, 16GB of DDR4 and an RTX 2070).

Even if not ideal, the current virtual scroller is still better than an almost half-minute freeze, imo.

Reply:
Because it completely wrecked the IntersectionObserver, I guess due to blocking caused by the heavy calculations it performs. I removed it so you guys actually can see how it works.

This PR will be marked as ready to review as soon as I get some opinions on what I mentioned in the main post (so I can keep working on this based on that) and the DOM recycling stuff ready and all the feedback around it (#275). Once that's done, not sure if in this PR or in another PR, this can be merged with little to no impact.

So consider this one quirk of the "Draft" nature of this PR :)